python/tk_desktop/desktop_engine_site_implementation.py

```diff
@@ -262,6 +263,18 @@ def run(self, splash, version, **kwargs):
         f.close()
         app.setStyleSheet(css)
 
+        # Initialize all of this after the style-sheet has been applied to any prompt are also
+        # styled after the Shotgun Desktop's visual-style.
+        splash.set_message("Initializing browser integration.")
+        try:
+            desktop_server_framework.launch_desktop_server()
```

Comment:
You can see [here](https://github.com/shotgunsoftware/tk-framework-desktopserver/blob/c67842ae14c75ef9ac3426add55fd5ea617e6787/framework.py#L30-L78) what this does.